src/Search.js

```diff
@@ -17,9 +18,9 @@ const Search = () => {
                 className="form-control"
                 placeholder="Customer name"
               />
-              <button className="btn btn-primary">Search</button>
             </div>
           </form>
+          <SearchButton />
```

Comment:
I don't think this will cause a big problem, but I just wanted to point out that the new `SearchButton` component has moved slightly from where it was before.